packages/react/src/Footer/Footer.tsx

```diff
@@ -19,4 +21,9 @@ const FooterRoot = forwardRef(({ children, className, ...restProps }: FooterProp
 
 FooterRoot.displayName = 'Footer'
 
-export const Footer = Object.assign(FooterRoot, { Bottom: FooterBottom, Top: FooterTop })
+export const Footer = Object.assign(FooterRoot, {
+  Bottom: FooterBottom,
+  Menu: FooterMenu,
+  MenuLink: FooterMenuLink,
+  Top: FooterTop,
```

Comment:
We decided on `Footer.Spotlight` for this.

Reply:
I'm not sure what you mean there. We did decide on `Footer.Spotlight` IIRC